.github/workflows/package.yml

```diff
@@ -12,13 +12,17 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [windows-2019, macos-latest, ubuntu-20.04]
```

Comment:
We can't do this; we need to build against the oldest version of a dynamically linked userspace possible, since glibc guarantees only backwards compatibility, not forwards compatibility. Applications linked against old Linux can run on new Linux, but applications linked against new Linux cannot run on old Linux.

Reply:
Gotcha. I think I can make everything work on 20.04.